### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by valerietanhx**

### CODE REVIEW

**Comment by sumhungyee** [comment]
LGTM, merged

File: docs/team/valerietanhx.md [hunk 1]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
+    This meant that much effort had to be put into ensuring that the parser could accurately detect when the two had been mixed,
+    regardless of the index(es) came before or after the date, and throw exceptions accordingly.
+    The delete command itself was also updated to use Optionals instead, as it now had to hold both indexes and dates,
+    but never at the same time. Much abstraction was required to keep the code clean and readable.
+* **New Feature:** Added the ability to delete all cases from a given date range.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear erroneous cases from the same date from the DengueHotspotTracker.
+  * Highlights: Similar to the previous feature, this enhancement was challenging as it required significant changes
+    to the architecture of the delete command parser and delete command. In addition to the aforementioned changes
+    to the parser and use of Optionals, new classes such as `Range`, `StartDate`, and `EndDate`, as well as
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:30]:
```suggestion
    to the parser and use of the `Optional` class, new classes such as `Range`, `StartDate`, and `EndDate`, as well as
```

File: docs/team/valerietanhx.md [hunk 2]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
+* **New Feature:** Added the ability to delete multiple indexes at once.
+  * Justification: This feature allows users to delete cases in batches, rather than having to do so one at a time.
+* **New Feature:** Added the ability to delete all cases from a given date.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear older or erroneous cases from the same date from the DengueHotspotTracker.
+  * Highlights: This enhancement was challenging as it required significant changes to the original delete command parser.
+    It caused the delete command to be the only command that could accept indexes and prefixes, _but never together_.
+    This meant that much effort had to be put into ensuring that the parser could accurately detect when the two had been mixed,
+    regardless of the index(es) came before or after the date, and throw exceptions accordingly.
+    The delete command itself was also updated to use Optionals instead, as it now had to hold both indexes and dates,
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:23]:
```suggestion
    The `delete` command itself was also updated to use the `Optional` class instead, as it now had to hold both indexes and dates,
```

File: docs/team/valerietanhx.md [hunk 3]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
+It is written in Java, and has about 9.5 kLoC.
 
 Given below are my contributions to the project.
 
-Code contributed: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=valerietanhx)
+* **New Feature:** Added the ability to delete multiple indexes at once.
+  * Justification: This feature allows users to delete cases in batches, rather than having to do so one at a time.
+* **New Feature:** Added the ability to delete all cases from a given date.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear older or erroneous cases from the same date from the DengueHotspotTracker.
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:18]:
```suggestion
    when the user wants to clear older or erroneous cases from the same date from the `DengueHotspotTracker`.
```

File: docs/team/valerietanhx.md [hunk 4]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
+  * Highlights: This enhancement was challenging as it required significant changes to the original delete command parser.
+    It caused the delete command to be the only command that could accept indexes and prefixes, _but never together_.
+    This meant that much effort had to be put into ensuring that the parser could accurately detect when the two had been mixed,
+    regardless of the index(es) came before or after the date, and throw exceptions accordingly.
+    The delete command itself was also updated to use Optionals instead, as it now had to hold both indexes and dates,
+    but never at the same time. Much abstraction was required to keep the code clean and readable.
+* **New Feature:** Added the ability to delete all cases from a given date range.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear erroneous cases from the same date from the DengueHotspotTracker.
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:27]:
```suggestion
    when the user wants to clear erroneous cases from the same date from the `DengueHotspotTracker`.
```

File: docs/team/valerietanhx.md [hunk 5]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
+  * Highlights: Similar to the previous feature, this enhancement was challenging as it required significant changes
+    to the architecture of the delete command parser and delete command. In addition to the aforementioned changes
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:29]:
```suggestion
    to the architecture of the `delete` command parser and `delete` command. In addition to the aforementioned changes
```

Comment (sumhungyee) [inline docs/team/valerietanhx.md:29]:
```suggestion
    to the architecture of the `delete` command parser and `delete` command. In addition to the aforementioned changes
```

File: docs/team/valerietanhx.md (hunk 1 above)
Comment (sumhungyee) [inline docs/team/valerietanhx.md:30]:
```suggestion
    to the parser and use of the `Optional` class, new classes such as `Range`, `StartDate`, and `EndDate`, as well as
```

File: docs/team/valerietanhx.md [hunk 6]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
 Given below are my contributions to the project.
 
-Code contributed: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=valerietanhx)
+* **New Feature:** Added the ability to delete multiple indexes at once.
+  * Justification: This feature allows users to delete cases in batches, rather than having to do so one at a time.
+* **New Feature:** Added the ability to delete all cases from a given date.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear older or erroneous cases from the same date from the DengueHotspotTracker.
+  * Highlights: This enhancement was challenging as it required significant changes to the original delete command parser.
+    It caused the delete command to be the only command that could accept indexes and prefixes, _but never together_.
```
Comment (sumhungyee) [inline docs/team/valerietanhx.md:20]:
```suggestion
    It caused the `delete` command to be the only command that could accept indexes and prefixes, _but never together_.
```

**Review by sumhungyee — APPROVED** [review]
LGTM, add suggestions if you wish.

File: docs/team/valerietanhx.md [hunk 7]
```diff
@@ -5,8 +5,41 @@ title: Valerie Tan's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
-Code contributed: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=valerietanhx)
+* **New Feature:** Added the ability to delete multiple indexes at once.
+  * Justification: This feature allows users to delete cases in batches, rather than having to do so one at a time.
+* **New Feature:** Added the ability to delete all cases from a given date.
+  * Justification: This feature allows users to delete all cases from a given date at once, which may be useful
+    when the user wants to clear older or erroneous cases from the same date from the `DengueHotspotTracker`.
+  * Highlights: This enhancement was challenging as it required significant changes to the original delete command parser.
```
Comment (valerietanhx) [inline docs/team/valerietanhx.md:19]:
```suggestion
  * Highlights: This enhancement was challenging as it required significant changes to the original `delete` command parser.
```

**Review by Tohtoroo — APPROVED** [review]

**Pull request merged**